contribute.md

```diff
@@ -4,7 +4,7 @@ title: Contribute
 description: Find out how to get involved in Bitcoin design
 permalink: /contribute/
 main_nav: true
-nav_order: 5
+nav_order: 2
 main_classes: -no-top-padding
```

Comment:
Feels a bit weird to have a reference to a CSS class in here? Can it be moved to the Guide layout at all?